lib/sles4sap_publiccloud.pm

```diff
@@ -368,17 +368,26 @@ sub stop_hana {
     record_info("Stopping HANA", "CMD:$cmd");
     if ($args{method} eq "crash") {
         # Crash needs to be executed as root and wait for host reboot
+
+        # Ensure the remote node is in a normal state before to trigger the crash
         $self->{my_instance}->wait_for_ssh(timeout => $timeout);
-        $self->{my_instance}->run_ssh_command(cmd => "sudo su -c sync", timeout => "0", %args);
-        # Try only extending ssh_opts
-        my $ssh_opts = $self->{my_instance}->ssh_opts . ' -o ServerAliveInterval=2';
+
+        $self->{my_instance}->run_ssh_command(cmd => "sudo su -c sync", timeout => $timeout);
```

Comment:
This is the only line that change.

Reply:
This is awesome. That command should return quickly (usually `sync` is almost immediate as it does most of its work in background), so it makes sense for our test code to wait for it to finish before continuing.